cmd/util/ledger/migrations/migrator_runtime.go

```diff
@@ -170,3 +178,13 @@ func NewInterpreterMigrationRuntime(
 		ContractNamesProvider:   env,
 	}, nil
 }
+
+type dummyHasher struct{ size int }
+
+func newDummyHasher(size int) hash.Hasher               { return &dummyHasher{size} }
+func (d *dummyHasher) Algorithm() hash.HashingAlgorithm { return hash.UnknownHashingAlgorithm }
+func (d *dummyHasher) Size() int                        { return d.size }
+func (d *dummyHasher) ComputeHash([]byte) hash.Hash     { return make([]byte, d.size) }
+func (d *dummyHasher) Write([]byte) (int, error)        { return 0, nil }
+func (d *dummyHasher) SumHash() hash.Hash               { return make([]byte, d.size) }
```

Comment:
Do these have to allocate new slices?

If the size field is kept, would it be possible to allocate once and return the same slice each time?

Given my comment above that the size can maybe be specialized to 0, could these functions just return `nil`?